src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -650,3 +663,32 @@ file to obtain scanning parameters. \_epi files can be a 3D or 4D - in the
 latter case all timepoints share the same scanning parameters. To indicate which
 run is intended to be used with which functional or diffusion scan the
 IntendedFor field in the JSON file should be used.
+
+#### B1+ fieldmaps 
```

Comment:
@Gilles86 these additions  look great! Should we go ahead and define some B1 mapping methods (e.g. double angle for B1+) in the suffix table, or would you rather prefer to do it later?

Reply:
Sure. I have very little expertise about this though!